go/cmd/aggregate-crls/crl-auditor.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"net/url"
+	"sync"
+	"time"
+
+	"github.com/jcjones/ct-mapreduce/storage"
+)
+
+var (
+	AuditKindFailedDownload     CrlAuditEntryKind = "Failed Download"
+	AuditKindFailedProcessLocal CrlAuditEntryKind = "Failed Process Local"
+	AuditKindFailedVerify       CrlAuditEntryKind = "Failed Verify"
+	AuditKindFailedVerifyLocal  CrlAuditEntryKind = "Failed Verify Local"
```

Comment:
It might be worth a comment to describe the difference between "Verify" and "Verify Local".

Reply:
mmm yeah, I don't need a separate enum for this. Thanks!